src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -27,6 +27,76 @@ public sealed partial class ApplicationIdentity : System.Runtime.Serialization.I
     }
 }
 
+namespace System.Data.Common
+{
+    public abstract partial class DBDataPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        [Obsolete]
```

Comment:
@weshaggard do you care if we delete [obsolete(msg,true)] members from netcoreapp? you already did for netstandard so I assume that's OK